README.md

```diff
@@ -1,2 +1,47 @@
-# workflows
-Reusable workflows for use in the Fastify organization
+# Workflows
+
+Reusable workflows for use in the Fastify organization.
+
+## Intro
+
+GitHub [introduced reusable workflows](https://github.blog/2021-11-29-github-actions-reusable-workflows-is-generally-available/) on 2021-11-29 which, as the name suggests, are workflows that can be referenced across the entirety of GitHub.
+
+For more information, including limitations, [see the GitHub Docs](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows).
+
+## Usage
+
+A reusable workflow is called by using the `uses` keyword in another workflow:
+
+```yml
+name: CI
+
+on:
+    push:
+        branches:
+            - main
+            - master
```

Comment:
I would remove these or the GHA will not run when a PR is updated

Reply:
Whoops, sorry those are from my own CI workflows, forgot Fastify doesn't restrict them to branch.

Reply:
Fixed by https://github.com/fastify/workflows/pull/1/commits/7a665456b4df201c42cec482c950cbd8dcccba58